R/rcodesign/build_tarballs.jl

```diff
@@ -24,35 +23,36 @@ if [[ "${target}" == x86_64-apple-darwin* ]]; then
     cp -ra System "/opt/${target}/${target}/sys-root/."
     popd
 fi
-cd $WORKSPACE/srcdir/PyOxidizer-apple-codesign-*/apple-codesign/
-cargo build --release
-install -Dvm 755 "../target/${rust_target}/release/rcodesign${exeext}" "${bindir}/rcodesign${exeext}"
-install_license ../LICENSE
+
+cd $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/
```

Comment:
```suggestion
cd $WORKSPACE/srcdir/PyOxidizer-apple-codesign-*/apple-codesign/
```

Reply:
This change is here because I used the [original source](https://github.com/indygreg/apple-platform-rs) for the `rcodesign`.

Reply:
```suggestion
cd $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-*/
```